### PR TITLE
9824 686 chapter 4

### PR DESCRIPTION
**Pull request by lbai001**
screencapture-localhost-3001-disability-benefits-686-dependent-status-unmarried-children-2018-05-14-06_03_48
screencapture-localhost-3001-disability-benefits-686-dependent-status-unmarried-children-2018-05-14-06_04_02
screencapture-localhost-3001-disability-benefits-686-dependent-status-unmarried-children-address-0-2018-05-14-06_03_13
screencapture-localhost-3001-disability-benefits-686-dependent-status-unmarried-children-information-0-2018-05-14-06_03_29

### CODE REVIEW

File: src/applications/disability-benefits/686/config/form.js [hunk 1]
```diff
@@ -511,6 +526,230 @@ const formConfig = {
           }
         }
       }
+    },
+    unMarriedChildren: {
+      title: 'Veteran\'s Unmarried Children',
```
Comment (jbalboni) [inline src/applications/disability-benefits/686/config/form.js]: We try to use `’` for all apostrophes.

File: src/applications/disability-benefits/686/config/form.js [hunk 2]
```diff
@@ -511,6 +526,230 @@ const formConfig = {
...
+                items: {
+                  type: 'object',
+                  required: ['childInHousehold'],
+                  properties: {
+                    childInHousehold: dependents.items.properties.childInHousehold,
+                    childInfo: {
+                      type: 'object',
+                      properties: {
+                        childAddress: address.schema(fullSchema686),
+                        personChildLiveWith: {
```
Comment (jbalboni) [inline src/applications/disability-benefits/686/config/form.js]: I think this is called `personWhoLivesWithChild` in the schema.

**Review by jbalboni — APPROVED** [review]
I have a couple of minor comments, but it looks good overall.

**Review by peggygannon — CHANGES_REQUESTED** [review]
@lbai001 Hi, Annie, these screens don't entirely match Judy's Invision Prototype: https://projects.invisionapp.com/share/M4H8VL3TY36#/screens/291249508

Some examples,
-- we changed Name fields to say: Child's first name, Child's last name, etc.. 
--Add Another Dependent button was changed to say Add Another Child
--the Child status labels are also slightly different. I edited them in the prototype to be more user friendly and plain language

cc @jbalboni

**Comment by lbai001** [comment]
@peggygannon 
screencapture-localhost-3001-disability-benefits-686-dependent-status-unmarried-children-2018-05-15-09_14_26
screencapture-localhost-3001-disability-benefits-686-dependent-status-unmarried-children-address-0-2018-05-15-09_15_08
screencapture-localhost-3001-disability-benefits-686-dependent-status-unmarried-children-information-0-2018-05-15-09_14_38

**Review by peggygannon — COMMENTED** [review]
Thanks @lbai001 001 Screens look good. 
It looks like the updated screen is missing the label for:
--Child is 18 to 23 years old and in school
I wanted to make sure that that label is in there? 
Could you let me know? thanks..

**Comment by lbai001** [comment]
@peggygannon the child inputed is not 18 -23 years old, birthday inputed is 2013.

here's a case for a 18 year old child:

screen shot 2018-05-15 at 10 52 52 am

**Review by peggygannon — APPROVED** [review]

**Comment by peggygannon** [comment]
@lbai001 Ok, thanks. Looks good.

**Pull request merged**